.prettierrc

```diff
@@ -2,5 +2,9 @@
   "semi": true,
   "singleQuote": true,
   "trailingComma": "es5",
-  "printWidth": 100
-}
+  "tabWidth": 2,
+  "printWidth": 115,
```

Comment:
Why change this to 115?

Reply:
It's closer to Beautify formatted file.
The difference from 100 to 115:
![image](https://user-images.githubusercontent.com/87154023/181626372-2a73c05c-a800-454f-b429-33b53d750049.png)